script.js

```diff
@@ -141,7 +141,7 @@ function initializeLibrary() {
   });
 
   // Sets the order to 0 if a category has no order property
-  if (!categories[0].hasOwnProperty('order')) categories[0].order = '0';
+  //if (!categories[0].hasOwnProperty('order')) categories[0].order = '0';
```

Comment:
@Animeboynz this actually breaks things because categories can have `order` 0 and thus `order` property doesn't get saved in backup. Something like should work.
```js
if (categories[0] && !categories[0].hasOwnProperty('order')) categories[0].order = '0';
```

Reply:
True lemme fix that